package.json

```diff
@@ -38,9 +38,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
     "lint": "eslint src --ext js,jsx",
     "lint:fix": "npm run lint -- --fix",
+    "prettier": "prettier \"src/**/*.{jsx,css,js}\" --write",
+    "prettier:check": "npm run prettier --check",
```

Comment:
It's a bit weird to have both `--check` and `--write`. I would do 2 different commands.